### ISSUE TITLE
Email Styles Not Included in Staging

### ISSUE THREAD

**Issue by mzagaja** [MAPC/youth-match-v2#126]
In the staging environment the sent email only included the font face style:

```
<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv=3D"Content-Type" content=3D"text/html; charset=3Dutf=
-8">
    =

    =

  <style>@font-face {
font-family: 'Source Sans Pro'; font-style: normal; font-weight: 400; src=
: local('Source Sans Pro'), local('SourceSansPro-Regular'), url(https://f=
onts.gstatic.com/s/sourcesanspro/v9/ODelI1aHBYDBqgeIAH2zlNzbP97U9sKh0jjxb=
PbfOKg.ttf) format('truetype');
}
</style>
</head>
```

Not sure why that is since premailer-rails suggests it should include all of it. I had to make a tweak to get the assets to work in staging though probably due to following the direction of https://github.com/rails/sprockets-rails/issues/237#issuecomment-231940623 in the commit  018a21ca6e1afe770d67bbae6c6e4750fedd8165. Need to find a new workaround.

**Comment by allthesignals** [comment]
Do the emails look styled? Generally there isn't supposed to be a lot in those style tags, as premailer-rails inlines everything. Email clients ignore the style tag anyway.

**Comment by mzagaja** [comment]
Not really. No buttons or anything. I forwarded you a copy. I think this could just be a rabbit hole of email that isn't worth fighting at this time, but if an easy fix is found then we can try.

**Comment by allthesignals** [comment]
Gotcha. I'm not sure that fix is working then, in commit 018a21c, unless you saw it working in development.

**Comment by mzagaja** [comment]
The update didn't work, but we're going live anyways. Thanks for the try though!
